.circleci/config.yml

```diff
@@ -6,6 +6,75 @@ jobs:
       - image: ubuntu:16.04
     steps:
       - run:
-          name: No-op.
-          command: echo "This command did nothing. It exists to configure CircleCI so that PRs that supply actual configuration will work."
-    working_directory: /var/code/googleapis
+          name: Add command to be able to trigger jobs.
+          command: |
+            touch trigger_job.sh && chmod a+x trigger_job.sh
+            echo >> trigger_job.sh <<EOF
+            #!/bin/bash
+            # More information about this script: http://bit.ly/2pn9lX8
+            job_name=$1
+            curl -u ${CIRCLE_API_TOKEN}: \
+              -d build_parameters[CIRCLE_JOB]=${job_name} \
+              -d revision=$CIRCLE_SHA1 \
+              https://circleci.com/api/v1.1/project/github/your_org/your_project/tree/$CIRCLE_BRANCH
```

Comment:
replacing `your_org/your_project` should fix it:
`https://circleci.com/api/v1.1/project/github/$CIRCLE_PROJECT_USERNAME/CIRCLE_PROJECT_REPONAME/tree/$CIRCLE_BRANCH`